### ISSUE TITLE
Marking of mandatory fields not always accurate

### ISSUE THREAD

**Issue by rolfkleef** [younginnovations/aidstream-old#72]
It seems that mandatory fields are not always marked as such, and some fields marked mandatory are accepted without input. It was shared as a trick in our workshop: just submit the form with one field entered, to get actual feedback on which fields are required.

(Sorry, I don't have a scenario to reproduce at hand yet...)

**Comment by stevieflow** [comment]
I think this applies to all the over-ride (`tied-status` etc) `transaction`.  @iatisupport and @jonihillman may have seem similar

**Comment by anjesh** [comment]
I think i got you @rolfkleef If you look at the following screenshot, TransactionType is mandatory field and hence "TransactionTypeCode" is also mandatory. However in case of ProviderOrganisation, if you decide to fill one of its any elements, then "Text" is compulsory field. It's kind of confusing on how to show instructions text to make it clear. Suggestions?

![screen shot 2014-10-13 at 3 45 06 pm](https://cloud.githubusercontent.com/assets/120488/4611868/dcd90d6c-52bf-11e4-986a-30004524c7be.png)

**Comment by rolfkleef** [comment]
I don't have suggestions at this point, I'll check with @Jaspermid to confirm this is indeed the case where it happens.

**Comment by stevieflow** [comment]
Text - this will usually apply to the name of the organisation?  Maybe change Text to 

> Text (eg Name of organisation)

?

**Comment by anjesh** [comment]
Yes @stevieflow i also see the areas where we should revisit the labels from what the standard says.
